src/daft-sql/src/python.rs

```diff
@@ -35,14 +35,29 @@ impl SQLFunctionStub {
 }
 
 #[pyfunction]
-pub fn plan_sql(
+pub fn sql_exec(
     sql: &str,
     session: &PySession,
     config: PyDaftPlanningConfig,
-) -> PyResult<PyLogicalPlanBuilder> {
-    let sess = Rc::new(session.into());
-    let plan = SQLPlanner::new(sess).plan_sql(sql)?;
-    Ok(LogicalPlanBuilder::new(plan, Some(config.config)).into())
+) -> PyResult<Option<PyLogicalPlanBuilder>> {
```

Comment:
suggestion: instead of returning an `Option`, what about introducing a new type for the statement output. I think eventually other statements will return outputs other than `LogicalPlan | None`. We could also just add that later when we need it.

Reply:
I was thinking about that, it would be nice for "CREATE" statements to return the statement, just like they would with the python DDL APIs from https://github.com/Eventual-Inc/Daft/pull/3902

I can make this an enum in Rust, but I would prefer to just use "object" on the python side so customers don't have to unwrap it – i.e. `sess.sql(..) -> object` where object could be DataFrame, Table, None, etc. Does that make sense to you too?